chia/clvm/spend_sim.py

```diff
@@ -187,8 +186,7 @@ async def close(self) -> None:
         await self.db_wrapper.close()
 
     async def new_peak(self) -> None:
-        # Create a protocol to make BlockRecord and SimBlockRecord interchangeable.
-        await self.mempool_manager.new_peak(self.block_records[-1], None)  # type: ignore[arg-type]
```

Comment:
you could simplify `benchmarks/mempool.py` as well

Reply:
Done.